### PR TITLE
Added feature

### PR DESCRIPTION
**Pull request by dchang0413**
Improved convert(c1,c2) search result
Added attabsorb
Added absorbnull
Added combo(n)
Added shield(n)

Also reduced refresh from 12 hours to 4 hours

### CODE REVIEW

File: padsearch/padsearch.py [hunk 1]
```diff
@@ -31,6 +28,9 @@
 * shuffle     : Board shuffle (aka refresh)
 * unlock      : Orb unlock
 * delay(n)    : Delay enemies by n
+* attabsorb   : Attribute Absorb shield null
+* absorbnull  : Damage Abasorb shield null
```
Comment (nachoapps) [inline padsearch/padsearch.py]: ABASORB lul

File: padsearch/padsearch.py [hunk 2]
```diff
@@ -387,8 +418,29 @@ def __init__(self, lexer):
...
 
+        if self.combo:
+            text = 'increase combo count by {}'.format(self.combo)
+            self.filters.append(lambda m, t=text: t in m.search.active_desc)
+
         if self.convert:
-            text = '{} orbs to '.format(self.convert[0][0]) + '{}'.format((self.convert)[0][1])
+            text_from = self.convert[0][0]
+            text_to = self.convert[0][1]
+            self.filters.append(lambda m,
```
Comment (nachoapps) [inline padsearch/padsearch.py]: Sick line ending bro

File: padsearch/padsearch.py [hunk 3]
```diff
@@ -554,6 +608,10 @@ def _make_search_config(self, input):
 
         await self.bot.say(box(json.dumps(m.search, indent=2, default=lambda o: o.__dict__)))
 
+    def get_emojis(self):
```
Comment (nachoapps) [inline padsearch/padsearch.py]: whats this doing in here

File: padsearch/padsearch.py [hunk 4]
```diff
@@ -527,15 +579,17 @@ def __init__(self, bot):
...
+            msg.append('Matched {} monsters'.format(len(matched_monsters)))
+            if len(matched_monsters) > 10:
+                msg[page] += ' (limited to 10)'#  Page {}'.format(page+1)
+        n = 0
         for m in matched_monsters:
-            msg += '\n\tNo. {} {}'.format(m.monster_no_na, m.name_na)
-
-        await self.bot.say(box(msg))
+            msg[int(n/10)] += '\n\tNo. {} {}'.format(m.monster_no_na, m.name_na)
+            n += 1
```
Comment (nachoapps) [inline padsearch/padsearch.py]: Is this prep work for pagination? Because I don't see what this is adding at the moment.

Comment (dchang0413) [reply padsearch/padsearch.py]: Yes

File: padguide2/padguide2.py [hunk 5]
```diff
@@ -1132,8 +1132,8 @@ def color_txt_to_list(txt):
             txt = txt.replace('colors', ' ')
             txt = txt.replace('orbs', ' ')
             txt = txt.replace('orb', ' ')
-            txt = txt.replace('critical poison', 'mortal')
-            txt = txt.replace('mortal poison', 'mortal')
+            txt = txt.replace('critical poison', 'mortalpoison')
```
Comment (nachoapps) [inline padguide2/padguide2.py]: Probably don't need both of these cleanups now that we're controlling the text right? I'm sure it's uniformly just one of them but i'm lazy to check which one.

File: padsearch/padsearch.py [hunk 6]
```diff
@@ -1,560 +1,626 @@
...
+    'blue': 'water',
+    'b': 'water',
+    'green': 'wood',
+    'g': 'wood',
+    'l': 'light',
+    'd': 'dark',
+    'heart': 'heal',
+    'h': 'heal',
+    'p': 'poison',
+    'mp': 'mortalpoison',
```
Comment (nachoapps) [inline padsearch/padsearch.py]: Instead of using two letters for these, why not just m for mortal and o for bomb (that's what I changed dawnglare to do).

**Pull request merged**